scripts/zkai_sota_artifact_watchlist_gate.py

```
@@ -0,0 +1,894 @@
+        raise SotaWatchlistError("system inventory drift")
+    for row in systems:
+        if set(row) != expected_system_fields:
+            raise SotaWatchlistError(f"system field set mismatch for {row.get('system')}")
+        system = row["system"]
+        status = row["status"]
+        if status not in ALLOWED_STATUSES:
+            raise SotaWatchlistError(f"unknown status for {system}")
+        if row["checked_at"] != CHECKED_AT:
+            raise SotaWatchlistError(f"checked_at drift for {system}")
+        if not _is_https_url(row["primary_source"]):
+            raise SotaWatchlistError(f"primary source must be https URL for {system}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)